### PR TITLE
fix console docs nav

### LINKED ISSUE #5565: fix console docs navigation
## overview
i just experienced a ton of pain trying to navigate around the console docs and find what i was looking for. it seems to be written as a guide that someone would read in its entirety instead of a place folks would go to find the answer to a specific question

## proposed solution
- [x] remove the groupings of `Accounts & Organizations`, `Collaboration`, and `Extensions` and flatten those out, so each of those pages appear in the nav
- [x] update linking to other pages and remove any `next` links

@davidwrede, let me know what you think, happy to take on this work.

### PR DESCRIPTION
**Pull request by susanev**
### Proposed changes

fixes #5565

intent is to go from this...
current-console-docs-nav

to this...
future-console-docs-nav

- [x] remove 3 category landing pages
- [x] move content if needed that was on those landing pages
- [x] expose individual pages in left nav
- [x] fix refs
- [x] fix redirects
- [x] move plan information to index page

... in the future i think we should consider moving org roles to the org page, and stack permissions to the projects and stacks page. also is it weird that the saml information isn't with these docs? anyways, wont be taking any of that on in this pr.

### CODE REVIEW

File: content/docs/intro/console/organizations.md [hunk 1]
```diff
@@ -145,9 +149,4 @@ and Settings tabs.
...
 | Members | A list of active members of the Pulumi organization. |
 | Teams | A [Team Pro]({{< relref "/pricing" >}}) feature that provides a way to assign stack permissions to groups of organization members. |
-| Policies | Lists of organization policies and policy groups. Policies allow you to set guardrails to enforce best practices and compliance. |
-| Settings | Organization settings including subscription and payment information and history, stack permissions, and links to Pulumi's [continuous delivery guides]({{< relref "/docs/guides/continuous-delivery" >}}). |
-
-## Next Steps
-
-* [Organization Roles]({{< relref "/docs/intro/console/collaboration/organization-roles" >}})
+| Settings | A place for you to manage your organization settings including subscription and payment information and history, base permissions to all of the organization's stacks (including stack creation and deletion), and links to Pulumi's [continuous delivery guides]({{< relref "/docs/guides/continuous-delivery" >}}). |
```
Comment (susanev) [inline content/docs/intro/console/organizations.md:152]: need to fix this

File: content/docs/intro/console/organization-roles.md [hunk 2]
```diff
@@ -14,8 +18,7 @@ organization.
 > Organizations billed per stack have slightly different rules regarding membership.
 
 To become a member of a Pulumi organization, you must be invited by an existing Pulumi
-organization admin or you must submit a request to the organization admin for approval. In addition, depending on the [organization type]({{< relref "/docs/intro/console/accounts-and-organizations#organization-types" >}}), you must also be a member of the third-party organization or group backing the Pulumi
-organization.
+organization administrator, or you must submit a request to the administrator for approval. In addition, depending on the organization type, you must also be a member of the third-party organization or group backing the Pulumi organization.
```
Comment (susanev) [inline content/docs/intro/console/organization-roles.md:21]: need to fix

File: content/docs/intro/console/pulumi-button.md [hunk 3]
```diff
@@ -2,11 +2,18 @@
...
+    parent: console
+    weight: 10
 aliases:
  - /docs/reference/service/pulumi-button/
  - /docs/console/extensions/pulumi-button/
+ - /docs/intro/console/extensions/pulumi-button/
 ---
 
+If you want to share a common cloud application, you can create a reusable template and share it with other developers using a “Deploy with Pulumi” Button.
+
```
Comment (susanev) [inline content/docs/intro/console/pulumi-button.md:16]: need to fix

File: content/docs/tutorials/aws/aws-ts-pulumi-webhooks.md [hunk 4]
```diff
@@ -84,7 +84,7 @@ After cloning this repo, run these commands from the working directory:
     pulumi stack output url
     ```
 
-1. Create a [Pulumi webhook](https://www.pulumi.com/docs/intro/console/extensions/webhooks/). Use the output from the previous step as the `Payload URL`.
+1. Create a [Pulumi webhook](https://www.pulumi.com/docs/intro/console/webhooks/). Use the output from the previous step as the `Payload URL`.
```
Comment (susanev) [inline content/docs/tutorials/aws/aws-ts-pulumi-webhooks.md:87]: need to fix

**Comment by cnunciato** [comment]
@susanev Now that https://github.com/pulumi/docs/pull/5494 has merged, would you mind resubmitting this on https://github.com/pulumi/pulumi-hugo? Sorry the the inconvenience and happy to help if you need.

**Comment by susanev** [comment]
yup sure, ill try on my own first. but will let ya know if end up needing help.

**Pull request closed without merging**